rfcs/023-images-endpoint/README.md

```diff
@@ -0,0 +1,52 @@
+# RFC 023: Images endpoint
+
+**Status:** :construction: Draft :building_construction:
+
+**Last updated:** 27/04/2020
+
+## Context and requirements
+
+- There is a new entity in the catalogue API: `Image`. These are conceptual, like works, and consist of images from the visual collections and interesting images from inside books and other materials.
+- We want to be able to search for images, and to fetch specified images.
+- The images endpoint should be consistent with the works endpoint.
+
+## Proposal
+
+The initial images endpoint will be in **beta**, and therefore will be liable to have breaking changes made without a version bump.
+
+### Image display model
+
+```
+{
+  "id": String,
+  "location": DigitalLocation,
+  "source": {
+    "id": String,
+    "sourceType": "Work"
```

Comment:
This should be `type` not `sourceType`.

Reply:
Are you sure? I think that might conflict with the ontology types / discriminators in the existing model - isn't the pattern that here we'd have `"type": "Source"`? `sourceType` is more like an enum of possible sources

Reply:
That's my point. It should be the ontology type. I don't think we need a `Source` type that just wraps the possible sources. The list of possible sources is other types in our model, ie `Work` mainly but possibly `Exhibition`, `Story`.